frontend/src/components/SearchBar/FilterBar.tsx

```diff
@@ -46,7 +46,7 @@ const FilterBar = <T extends object = {}>(props: PropsWithChildren<IProps<T>>) =
       }}
     >
       {({ isSubmitting, handleReset, setFieldValue }) => (
-        <Form>
+        <Form autoComplete="off">
```

Comment:
autocomplete disabled on the map filter bar, as it conflicts with geocoder and our autocomplete selects.